### ISSUE TITLE
Gem dependency conflict with Digital Ocean gem

### ISSUE THREAD

**Issue by ThatGerber** [hirakiuc/tinybucket#87]
```
    ...
      droplet_kit (= 1.4.1) was resolved to 1.4.1, which depends on
        faraday (~> 0.9.1)

    ...
      tinybucket (~> 1.0) was resolved to 1.0.0, which depends on
        faraday (= 0.9.0)
```

I'm unable to use gem on my project because of the strict dependency tinybucket places the Faraday middlware. Is there a reason to have it to tightly limited?

**Comment by hirakiuc** [comment]
Thanks for your report !

I have no reason to keep dependency with faraday 0.9.0.

I'll update faraday dependency.

**Comment by hirakiuc** [comment]
Merged pull request #90 , and released v1.3.0.

Please try v1.3.0 !
